src/localize/languages/en.json

```diff
@@ -9,7 +9,7 @@
     "reload": "Reload",
     "remove": "Remove",
     "repositories": "Repositories",
-    "repository": "Repository",
+    "repository": "View source code",
```

Comment:
This is used in the add custom repository dialog as well where this update will not work.